test/views/html/estimatedIncomeTax/ComplexEstimatedIncomeTaxViewSpec.scala

```diff
@@ -39,11 +39,17 @@ class ComplexEstimatedIncomeTaxViewSpec extends TaiViewSpec {
     behave like pageWithBackLink
 
     "have a heading for the Total estimated Income" in {
-      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.totalEstimatedIncome.subheading") + " £48,000")
+      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.totalEstimatedIncome.subheading"))
+    }
+    "have a paragraph for the Total estimated Income value " in {
+      doc(view) must haveParagraphWithText(messages("£48,000"))
     }
 
     "have a heading for the Income tax estimate" in {
-      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.incomeTaxEstimate.subheading") + " £15,000")
+      doc(view) must haveH2HeadingWithText(messages("tai.incomeTax.incomeTaxEstimate.subheading"))
+    }
+    "have a paragraph for the Income tax estimate value " in {
+      doc(view) must haveParagraphWithText(messages("£15,000"))
```

Comment:
Dont think this needs to be wrapped in a messages

Reply:
Yes 👍